catalog/internals/graphql-codegen/quilt-graphql.js

```diff
@@ -0,0 +1,137 @@
+const assert = require('assert')
```

Comment:
Probably, you can use ES6 imports if you rename file to `quilt-graphql.mjs`

Reply:
that was the plan, but it doesnt work -- codegen uses `require` to load plugins and node throws an exception saying smth like "use `import` to load mjs modules"